src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionCyclomaticComplexityTest.java

```diff
@@ -30,12 +30,18 @@
 
 public class XpathRegressionCyclomaticComplexityTest extends AbstractXpathTestSupport {
 
+    private final String checkName = CyclomaticComplexityCheck.class.getSimpleName();
```

Comment:
I'm surprised nothing is complaining that these fields can be static.

Reply:
I thought about it, persuaded myself to not do this, as thetr will be no benefits as tests are used in strict workflow, make it static will force it to be uppercased, so getter will return constant, ...

Reply:
All the other getters in src/test return a constant too.